eradiate/radprops/rad_profile.py

```diff
@@ -443,8 +443,7 @@ class US76ApproxRadProfile(RadProfile):
     )
 
     dataset = attr.ib(
-        default="spectra-us76_u86_4-4000_25711",
-        validator=attr.validators.in_({"spectra-us76_u86_4-4000_25711", "test"}),
+        default=None,
```

Comment:
Without a validator, you won't get a meaningful error message when the user will enter wrong values. How does debugging work in that case? I think you should, at least, validate for the data type.

Then have a look below: what happens at line 484 if I set `self.dataset` to some random non-string value? Or to a string value which is not a dataset ID?

Finally, please don't forget to update the docstring when you update attributes.

Reply:
> When you use the path resolver, don't forget that it's implemented as a singleton. See `solar_irradiance_spectral.py` for usage.

About the path resolver being a singleton, I have seen multiple instances of the `PathResolver` class, what do you mean then by that?

Reply:
Updated the docstrings!